eks/cluster/vpc.go

```diff
@@ -567,9 +601,19 @@ Outputs:
     Value: !Ref VPC
     Description: VPC ID
 
-  VPCCIDR:
-    Value: !Ref VPCCIDR
-    Description: VPCCIDR
+  VPCBlock1:
+    Value: !Ref VPCBlock1
+    Description: Primary VPC Block
+
```

Comment:
Remove this?

Reply:
Are you referring to the newline or the VPCBlock1?

Reply:
Empty new line